crates/iroha_cli/src/main.rs

```diff
@@ -354,22 +404,46 @@ mod blocks {
     pub struct Args {
         /// Block height from which to start streaming blocks
         height: NonZeroU64,
+
+        /// Wait timeout in seconds
```

Comment:
IMO, specifying this in seconds might not be enough granularity. How about making this milliseconds?

Reply:
I think either way is ok

Reply:
In `multisig` subcommand, `multisig::Register.transaction_ttl` uses `humantime::Duration`. I'd suggest reusing it here for consistency.